workflow/rocoto/gfs_tasks.py

```diff
@@ -24,6 +24,12 @@ def stage_ic(self):
 
         # Atm ICs
         if self.app_config.do_atm:
+           #The if block below is added for AWS.
+           #If we have a proper way to define 'BASE_CPLIC', this if block can be removed.
```

Comment:
`BASE_CPLIC` is defined in `hosts/awspw.yaml` and that gets filled into `config.base`.